test/views/ConfirmationCurrentPreviousViewSpec.scala

```diff
@@ -113,10 +115,57 @@ class ConfirmationCurrentPreviousViewSpec extends ViewBehaviours {
         claimAmount,
         scottishClaimAmountsRates.intermediateRate
       ))
+
       assertContainsText(doc, messages("confirmation.englandHeading"))
       assertContainsText(doc, messages("confirmation.scotlandHeading"))
     }
 
+    "display correct text based on claim amount increase" in {
+
+      val viewWithSpecificAnswers = applyView(npsAmount = 50, claimAmount = 1000)(fakeRequest, messages)
+
+      val doc = asDocument(viewWithSpecificAnswers)
+
+      assertContainsMessages(doc, messages("confirmation.personalAllowanceIncrease", 50, 1000))
+
+      assertDoesntContainMessages(doc,
+        messages("confirmation.personalAllowanceDecrease", 1000, 50),
+        messages("confirmation.newPersonalAllowance", 50)
+      )
```

Comment:
Isn't one side of this enough as it can only be one or the other?

Reply:
True, though it cant hurt to test that its hidden, this can be removed it preferred